brewtils/models.py

```diff
@@ -528,6 +529,7 @@ class RequestTemplate(BaseModel):
         "comment",
         "metadata",
         "output_type",
+        "garden",
```

Comment:
Hmm. Is this the target garden or the source garden? Maybe the name should specify?